docs/sources/write/front-matter/index.md

```diff
@@ -123,11 +151,13 @@ To test an alias results in the correct redirect, use your browser or a command-
    curl localhost:3002/docs/grafana/latest/panels/working-with-panels/
    ```
 
-   The output is similar to the following:
+````
 
-   ```console
-   <!doctype html><html><head><script>const destination="http://localhost:3002/docs/grafana/latest/panels-visualizations/panel-editor-overview/";console.log(window.location.search),document.head.innerHTML=`<meta http-equiv="refresh" content="0; url=${destination}${window.location.search}"/>`</script><title>http://localhost:3002/docs/grafana/latest/panels-visualizations/panel-editor-overview/</title><link rel=canonical href=http://localhost:3002/docs/grafana/latest/panels-visualizations/panel-editor-overview/><meta name=robots content="noindex"><meta charset=utf-8><noscript><meta http-equiv=refresh content="0; url=http://localhost:3002/docs/grafana/latest/panels-visualizations/panel-editor-overview/"></noscript></head></html>
-   ```
+The output is similar to the following:
+
+```console
+<!doctype html><html><head><script>const destination="http://localhost:3002/docs/grafana/latest/panels-visualizations/panel-editor-overview/";console.log(window.location.search),document.head.innerHTML=`<meta http-equiv="refresh" content="0; url=${destination}${window.location.search}"/>`</script><title>http://localhost:3002/docs/grafana/latest/panels-visualizations/panel-editor-overview/</title><link rel=canonical href=http://localhost:3002/docs/grafana/latest/panels-visualizations/panel-editor-overview/><meta name=robots content="noindex"><meta charset=utf-8><noscript><meta http-equiv=refresh content="0; url=http://localhost:3002/docs/grafana/latest/panels-visualizations/panel-editor-overview/"></noscript></head></html>
+```
```

Comment:
Not sure what's wrong here (because it looks like it's closed) but in the output, the rest of this page is included in this code block.

Reply:
Ah just bad auto-formatting on my part, I'll fix that in a commit.

Reply:
Fixed in 77e67df147fdd1e79272cac917b3d7fbc6d7df38